taskcluster/scripts/toolkit/compile_openssl.ps1

```diff
@@ -0,0 +1,47 @@
+$REPO_ROOT_PATH =resolve-path "$PSScriptRoot/../../../"
+$FETCHES_PATH =resolve-path "$REPO_ROOT_PATH/../../fetches"
+
+# Enter the DEV Shell
+. "$FETCHES_PATH/VisualStudio/enter_dev_shell.ps1"
+
+#New-Item -Path fetches -ItemType "directory"
+Set-Location -Path $FETCHES_PATH
+
+Invoke-WebRequest -Uri https://github.com/openssl/openssl/archive/refs/tags/OpenSSL_1_1_1m.zip -OutFile openssl.zip
+Invoke-WebRequest -Uri https://www.nasm.us/pub/nasm/releasebuilds/2.15.05/win64/nasm-2.15.05-win64.zip -OutFile nasm.zip
+
+Expand-Archive nasm.zip -DestinationPath .
+Expand-Archive openssl.zip -DestinationPath .
+
+Set-Location openssl-OpenSSL_1_1_1m
+
+$NASM_PATH = resolve-path ../nasm-2.15.05
+$env:PATH = "$env:PATH;$NASM_PATH"
+
+if(!(Test-Path ../SSL)){
+    New-Item -Path ../SSL -ItemType "directory"
+}
+# Means we put it $FETCHES_PATH/SSL
+$SSL_OUT_PATH = resolve-path ../SSL
+
+$PERL_PATH =resolve-path "$FETCHES_PATH/perl/bin"
+$env:PATH = "$PERL_PATH;$env:PATH"
+
+perl Configure  VC-WIN64A `
+                --release `
+                --prefix="$SSL_OUT_PATH" `
+                --openssldir="$SSL_OUT_PATH/SSL" `
+                no-tests `
+                no-engine `
+                -DOPENSSL_NO_ENGINE
+
```

Comment:
This will resolve --openssldir to `Z:/<random_task_id>/fetches/ssl`  - 
@mkestler-mozilla - Is this ok, DOPENSSL_NO_ENGINE should be enough or do will still require it to point a protected folder? 
In a tc-job, we don't get admin-rights so setting either to C:/<priviledged folder> would be a blocker :D

Reply:
This should be ok.